docker/README.md

```diff
@@ -3,46 +3,86 @@
+	  command: -dbhost mongo
+
+	mongo:
+	  image: mongo:2.6
+
+Then run
+
+	sudo docker-compose up
+
+This way is equivalent to the previous one, except that it pulls the image from the Docker Registry instead of building your own. Keep in mind though that everything is run locally. 
+
+> **Warning**
+> Everything you do with Orion Context Broker when dockerized is non-persistent. *You will lose all your data* if you turn off the MongoDB container. This happens with either this method or the manual one.
+> If you want to prevent this from happening take a look at [this link](https://registry.hub.docker.com/_/mongo/) in section *Where to Store Data* of the MongoDB docker documentation. In it you will find instructions and ideas on how to make your MongoDB data persistent.
 
 ## Build the image
 
-This is an alternative approach than the one presented in section [The Fastest Way](#the_fastest_way). You do not need to go through these steps if you have used docker-compose.
+This is an alternative approach to the one presented in the previous section. You do not need to go through these steps if you have used docker-compose. The end result will be the same, but this way you have a bit more of control of what's happening.
 
 You only need to do this once in your system:
 
-	docker build -t orion .
+	sudo docker build -t orion .
+
+> **Note**
+> If you don't want to have to use `sudo` in this or in the next section follow [these instructions](http://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo).
+
 
 The parameter `-t orion` gives the image a name. This name could be anything, or even include an organization like `-t org/fiware-orion`. This name is later used to run the container based on the image.
 
 If you want to know more about images and the building process you can find it in [Docker's documentation](https://docs.docker.com/userguide/dockerimages/).
     
-## Run the container
+### Run the container
+
+The following line will run the container exposing port `1026`, give it a name -in this case `orion1`, link it to a mongodb docker, and present a bash prompt. This uses the image built in the previous section.
 
-The following line will run the container exposing port `1026`, give it a name -in this case `orion1`, link it to a mongodb docker, and present a bash prompt.
+	  sudo docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 orion -dbhost mongodb
 
-	  docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 orion -dbhost mongodb
+If did not build the image yourself and want to use the one on Docker Hub use the following command:
+
+	  sudo docker run -d --name orion1 --link mongodb:mongodb -p 1026:1026 fiware/orion -dbhost mongodb
+
+> **Note**
+> Keep in mind that if you use this last command you get access to the tags and specific versions of Orion. For example, you may add `fiware/orion:0.22` if you need that particular version. If you don't specify a version you are pulling from `latest`.
 
```

Comment:
nit: avoid contractions do not instead of don't , etc. this comments applies to the rest of the code